firestore/src/android/wrapper.h

```diff
@@ -44,9 +45,10 @@ class Wrapper {
   Wrapper& operator=(Wrapper&& wrapper) = delete;
 
   FirestoreInternal* firestore_internal() { return firestore_; }
-  const jni::Object& ToJava() const { return obj_; }
-
-  static jni::Object ToJava(const FieldValue& value);
+  jni::Local<jni::Object> ToJava() const {
```

Comment:
Similarly, move Wrapper::ToJava() to the .cc file as well, since it is no longer a simple "getter" method.